lib/main_common.pm

```diff
@@ -548,4 +549,11 @@ sub load_filesystem_tests() {
     }
 }
 
+sub load_iso_in_external_tests() {
```

Comment:
The methods above mostly have paranthesis, some have not. Would adapt all according to your preferences (without I assume?)